#goal definition
int32 grippermove_cmd
int32 gripper_side
float64 gripper_desired_mm


---
#result definition
int32 gripper_side
float64 gripper_desired_mm
int32 result_flag

---
#feedback
int32 gripper_side
float64 gripper_desired_mm

